src/main/java/seedu/address/logic/commands/ScrollCommand.java

```diff
@@ -0,0 +1,38 @@
+package seedu.address.logic.commands;
+
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.ItemModel;
+
+/**
+ * Abstract class for scrolling.
+ */
+public abstract class ScrollCommand extends Command {
+
+    public static final String COMMAND_WORD = "down";
+    public static final String MESSAGE_SUCCESS = "Scrolling down...";
+    public static final String MESSAGE_USAGE = "Message Usage";
```

Comment:
Why is there a need for this three lines here?

Reply:
getCommandWord() needs the COMMAND_WORD

execute() needs MESSAGE_SUCCESS

i can delete message usage uh

Reply:
ok. i think when @lrchema changes the `getCommandWord()` to the UndoableCommand abstract class then we can remove it.

Reply:
I removed alr :)